std/Sys.hx

```diff
@@ -92,9 +92,13 @@ extern class Sys {
 	/**
 		Returns the type of the current system. Possible values are:
 		 - `"Windows"`
```

Comment:
Might as well fix the issue here – there needs to be an empty line between the text and the list (like [in `Array`](https://github.com/HaxeFoundation/haxe/blob/4.0.5/std/Array.hx#L198-L200)) for the list to actually render.